src/odinapi/views/level2.py

```diff
@@ -38,7 +39,9 @@
 
 
 DEFAULT_LIMIT = 1000
+DOCUMENT_LIMIT = 50000
```

Comment:
This looks to be the same as the `HARD_LIMIT`, don't define the same constant twice.

Reply:
both constants still exist.

Reply:
see above

Reply:
I don't understand the comment, they still both exists

Reply:
Ok, the comment above disappeared. There is a function in level2db module that is using hard limit, not covered by this PR. I will import parameter from here since we have the other parameters here.